apps/precinct-scanner/src/screens/PollWorkerScreen.tsx

```diff
@@ -164,6 +176,7 @@ const PollWorkerScreen = ({
               </Button>
             )}
           </p>
+          <p>Security Code: {securityCode}</p>
```

Comment:
I suggest we format this like 1Password does, i.e. `123·456`, unless there's a good reason not to.

Reply:
+1 to this, even just a space in between would be nice for readability.